packages/kit/src/core/dev/plugin.js

```diff
@@ -512,6 +512,9 @@ export const svelte = function (svelte_config) {
 			...svelte_config.compilerOptions,
 			hydratable: !!svelte_config.kit.browser.hydrate
 		},
+		hot: {
+			optimistic: false
+		},
```

Comment:
we can remove this once https://github.com/sveltejs/vite-plugin-svelte/pull/349 is released

```suggestion
		hot: {
			optimistic: false
		},
```